extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authentication/db/updater/CoordinatorBasicAuthenticatorMetadataStorageUpdater.java

```diff
@@ -156,6 +161,9 @@ public void start()
                 )
             );
           }
+
+          BasicAuthConfig config = new BasicAuthConfig(dbConfig);
```

Comment:
I started out on this path first of having a separate BasicAuthLDAPConfig but remember running into some issues that I can't specifically recall. I don't recall trying to inject it using guice, so I'll look into further. Great suggestion, thanks

Reply:
I've addressed this, and introduced BasicAuthLDAPConfig. But this line of code is now irrelevant since I've removed functionality related to storing and updating config.